### ISSUE TITLE
Scikit-Learn

### ISSUE THREAD

**Issue by pulasthi** [cloudmesh-community/book#242]
Coordinate with tyler to get a to do list in here and assign specific areas to different people ...

- [ ] Tyler: integarte mindmak
- [ ] Tyler: assign tasks to students
- [ ] Students identify tasks in this section, so tyler can assign them to you in this list
- [ ] Tyler, maybe some undergrads could be useful here too
- [ ] Fix punctuation no psces befor :

**Comment by anthonyduer** [comment]
Would like to work on this section if it's available. :)

**Comment by manjunathsivan** [comment]
hi Pulasthi, 
   Would like to add content to scikit learn chapter. Please let me know if it can be assigned.

Regards
Manjunath Sivan

**Comment by manjunathsivan** [comment]
Can I do a chapter on this. The current information is just a code snippet.

**Comment by laszewsk** [comment]
please coordinate with each other

**Comment by manjunathsivan** [comment]
Will be adding the chapter with the following sections
1. Introduction
2. Supervised learning
3. Unsupervised Learning
4. Linear Models
5. Support Vector Machines
6. Stochastic Gradient Descent
7. Nearest Neighbors
8. Feature Selection

**Comment by anthonyduer** [comment]
@manjunathsivan Hey Manjunath - I think we could also use a installation section after the Introduction. I'm freeing up soon so can start helping out. :)

**Comment by laszewsk** [comment]
Install

pip install numpy
pip install scipy
pip install scikit-learn

for us do not use conda install

**Comment by manjunathsivan** [comment]
hi Prof, 
   Can you please tag this as a chapter also and assign it to me.
Regards
Manjunath Sivan

**Comment by laszewsk** [comment]
already assigned

**Comment by laszewsk** [comment]
all those working on this should watch the checkins. There was some issue as a file was renamed. I let Tyler coordinate the rename if even needed. Also Tyler can you add the mindmap you draw to this section, than people coudl start doing a todo list and each can do part of that to do list. For example one could work on regression while another works on clustering

Important is that we do not plagiuarize, and plagiarized content will be immediatly deleted.

**Comment by laszewsk** [comment]
So before anyone submits ocntent make sure its not plagiarized, even if you just say this is just a draft. a draft can not be palgiarized either.

**Comment by laszewsk** [comment]
THis was supposed to develop OpenAPI based REST services while starting from the OpenAPI spec
